### PR TITLE
Updated docker-compose.yml configuration

### PR DESCRIPTION
**Pull request by Irkam**
[The --link option is deprecated in latest versions of Docker.](https://docs.docker.com/network/links/) 
This configuration makes use of the `bridge` network adapter.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,15 +4,11 @@ services:
         restart: always
         image: sharelatex/sharelatex
         container_name: sharelatex
-        depends_on:
```
Comment (prohde) [inline docker-compose.yml]: Why did you delete the depends_on?

**Comment by mserranom** [comment]
Closing, as this is not in a mergeable state (feel free to raise a new one @lrkam)
